packages/stargate/src/stargateclient.ts

```diff
@@ -139,7 +141,7 @@ export interface PrivateStargateClient {
 export class StargateClient {
   private readonly tmClient: Tendermint34Client | undefined;
   private readonly queryClient:
-    | (QueryClient & AuthExtension & BankExtension & StakingExtension & TxExtension)
+    | (QueryClient & AuthExtension & BankExtension & StakingExtension & TxExtension & SlashingExtension)
```

Comment:
I would only add the extensions that are needed for the higher level functions in the stargate client. The query client here is private and I think callers should create their own, depending on their needs and backend availabilities.